src/lua/uri.lua

```diff
@@ -231,6 +231,16 @@ local function format(uri, write_password)
     uribuf.path = uri.path
     uribuf.query = uri.query
     uribuf.fragment = uri.fragment
+    -- IPv6 addresses need to be output encapsulated in brackets []. Since
+    -- gh-9556, we must set the `host_hint` to determine the type of `uri.host`
+    -- (refer to `src/lib/uri/uri.c:uri_format`).
```

Comment:
Nit: I would refer to `uri_format`, but not gh-9556, because iti s already fixed issue for a reader of this code.

For example.

> The `uri_format` function needs the hint to enclose an IPv6
> address into square brackets.

Reply:
comment corrected